src/Realm.C

```diff
@@ -545,13 +550,37 @@ Realm::look_ahead_and_creation(const YAML::Node & node)
     dataProbePostProcessing_ =  new DataProbePostProcessing(*this, *foundProbe[0]);
   }
 
-  // look for ActuatorLine
-  std::vector<const YAML::Node *> foundActuatorLine;
-  NaluParsingHelper::find_nodes_given_key("actuator_line", node, foundActuatorLine);
-  if ( foundActuatorLine.size() > 0 ) {
-    if ( foundActuatorLine.size() != 1 )
+  // look for Actuator
+  std::vector<const YAML::Node*> foundActuator;
+  NaluParsingHelper::find_nodes_given_key("actuator", node, foundActuator);
+  if ( foundActuator.size() > 0 ) {
+    if ( foundActuator.size() != 1 )
       throw std::runtime_error("look_ahead_and_create::error: Too many actuator line blocks");
-    actuatorLine_ =  new ActuatorLine(*this, *foundActuatorLine[0]);
+
+    if ( (*foundActuator[0])["actuator"]["type"] ) {
+      const std::string ActuatorTypeName = (*foundActuator[0])["actuator"]["type"].as<std::string>() ;
+      switch ( ActuatorTypeMap[ActuatorTypeName] ) {
+      case ActuatorType::ActLinePointDrag : {
+	actuator_ =  new ActuatorLinePointDrag(*this, *foundActuator[0]);
+	break;
+      }
+#ifdef NALU_USES_OPENFAST
+      case ActuatorType::ActLineFAST : {
+	actuator_ =  new ActuatorLineFAST(*this, *foundActuator[0]);
+	break;
+      }
```

Comment:
Add `#else` here to warn users that OpenFAST integration was requested but was not enabled during compile time?